tests/test_processing.py

```diff
@@ -0,0 +1,37 @@
+"""Unittests for gray.processing."""
```

Comment:
Rewrite tests in pytest, please. That's not very convenient to have both unittest and pytest in the same project.

Reply:
Yep, `unittest.mock` is totally fine